src/lib/get-messages.js

```diff
@@ -0,0 +1,11 @@
+import assign from 'object-assign';
+import locales from '../../public/locales.json';
+
+function getMessages(locale) {
+  if (['es-AR', 'es-CL', 'es-MX', 'es-XL'].indexOf(locale) >= 0) {
```

Comment:
note to self: create folders for es-* with .keep files in it, since we will use /es only here too